lib/kamal/configuration/env.rb

```diff
@@ -1,18 +1,22 @@
 class Kamal::Configuration::Env
-  attr_reader :secrets_keys, :clear, :secrets_file
+  attr_reader :secrets_keys, :clear, :secrets_file, :env_file
   delegate :argumentize, to: Kamal::Utils
 
   def self.from_config(config:, secrets_file: nil)
-    secrets_keys = config.fetch("secret", [])
-    clear = config.fetch("clear", config.key?("secret") || config.key?("tags") ? {} : config)
+    env_key_config = config.class == Kamal::Configuration ? config.env : config.fetch("env", {})
```

Comment:
The "config" variable name is not too clear to what this represents. This is not "config", this is the "env" key from config with this values. Maybe we can also extract this to his own PR also?